udata_front/theme/gouvfr/info.json

```diff
@@ -0,0 +1,13 @@
+{
+  "application": "udata",
+  "identifier": "gouvfr",
+  "name": "Data.gouv.fr",
+  "author": "Etalab",
+  "description": "Data.gouv.fr theme",
+  "website": "http://data.gouv.fr",
+  "license": "AGPL",
+  "version": "0.1.0",
+  "doctype": "html5",
+  "variants": ["deuil"],
+  "github_faq_url_pattern": "https://github.com/etalab/udata-gouvfr/edit/master/udata_front/templates/faq/{page_name}.html"
```

Comment:
This path is not leading to anything. I couldn't find any usage of `github_faq_url_pattern` in the code (whereas the other fields are loaded in [flask_themes2](https://flask-themes2.readthedocs.io/en/latest/#info-json-fields). Any idea if we should keep this setting @abulte?

Reply:
I think it's been dead a long time, RIP.